inc/admin.php

```diff
@@ -43,6 +43,10 @@ public function __construct() {
 		}
 		add_action( 'optml_after_setup', [ $this, 'register_public_actions' ], 999999 );
 
+		if ( ! function_exists( 'is_wpcom_vip' ) ) {
+			add_filter( 'upload_mimes', [ $this, 'allow_meme_types' ] ); // phpcs:ignore WordPressVIPMinimum.Hooks.RestrictedHooks.upload_mimes
+			add_filter( 'wp_check_filetype_and_ext', [ $this, 'fix_mime_type_svg' ], 75, 4 );
```

Comment:
I think this is redundant from my testing the attached function already receives the correct type when it is called.

Reply:
Wasn't working for me locally without it but works in other instances.